models/Mongoschema.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const questionnaireSchema = new Schema(
+  {
+    date: { type: Date },
+    sleep: {
+      quality: {
+        type: String,
+        enum: ['terrible', 'bad', 'neutral', 'satisfied', 'excelent'],
+        required: true,
+      },
+      quantity: {
+        goToBed: { type: String, required: true },
+        wakeUp: { type: String, required: true },
+        total: { type: Number, required: true },
+      },
+    },
+    trainning: {
+      time: { type: String },
+    },
+    organizationTime: {
+      tool: {
+        type: [String],
+        enum: ['app', 'mobile', 'schedule', 'notes'],
+      },
+    },
+    screenUsage: {
+      beforeSleep: {
```

Comment:
Add more validations:
1. it should be required
2. response message when validations are wrong